membership-attribute-service/app/services/IdentityAuthService.scala

```diff
@@ -71,22 +71,25 @@ class IdentityAuthService(identityPlayAuthService: IdentityPlayAuthService)(impl
         // Something unexpected
         case Left(other) => IO.raiseError(other)
         // Request has valid Okta token
-        case Right((credentials: OktaUserCredentials, user)) => IO.pure(Right(UserAndCredentials(user, credentials)))
+        case Right((credentials: OktaUserCredentials, user: OktaAuthenticatedUserInfo[MDAPIIdentityClaims, UserFromToken])) =>
+          IO.pure(Right(UserAndCredentials(user.localAccessTokenClaims, credentials)))
         // Request has valid Idapi credentials
-        case Right((credentials: IdapiUserCredentials, user)) => IO.pure(Right(UserAndCredentials(user, credentials)))
+        case Right((credentials: IdapiUserCredentials, user: IdapiAuthenticatedUserInfo)) =>
+          IO.pure(Right(UserAndCredentials(UserFromToken.fromIdapiUser(user.idapiUser), credentials)))
```

Comment:
seemed to be a bit more unpacking needed - rather than getting back a UserFromToken, we got either a basic IDAPI User or a more complex pair of okta objects based on the implicits.  Then we deal with it here accordingly.